amundsen-gremlin: first main commit.

friendtocephalopods opened the pull request:
Add bulk loader, script translator. Tests.

Add commit/push hooks for style and to prevent credential leakage.

TODO: update amundsencommon version

alran commented:
This lgtm! The only thing that stands out to me is that I think it would be good to add a fairly robust readme to this PR that explains what someone needs to do to wire this up for their use case.

friendtocephalopods committed:
amundsen-gremlin: first main commit.

Add bulk loader, script translator, streams. Tests.

Add commit/push hooks for style and to prevent credential leakage.
Add github checks and workflow.

Signed-off-by: Joshua Hoskins <hoskins@squareup.com>

friendtocephalopods committed:
feat: add github workflows

Signed-off-by: Joshua Hoskins <hoskins@squareup.com>

friendtocephalopods committed:
Merge pull request #2 from friendtocephalopods/add-workflow

feat: add github workflows
